content/how-to-write-code/reading/comments/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Comments"
+date: 2023-07-13T14:00:03-05:00
+draft: false
+weight: 3
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+As programs get bigger and more complicated, they get more difficult to read.
+Good programmers try to make their code understandable to others, but it is
+still tricky to look at a large program and figure out what it is doing and
+why.
+
+Best practice encourages us to add notes to our programs, which clearly
+explain what the program is doing. These notes are called *comments*.
+
+A **comment** is text within a program intended only for a human reader---it is
+completely ignored by the compiler or interpreter. In JavaScript, the `//`
+token starts a comment, and the rest of the line gets ignored. For comments
+that stretch over multiple lines, the text falls between the symbols
+`/*   */`.
+
+## Experiment with Comments
+
+1. On your own computer, in `javascript-web-dev-projects`, locate the `how-to-write-code` folder. Within that folder, you will find an example called `Comments`. *You should use the terminal to do so.* Here are the steps:
```

Comment:
I may be missing something, but I don't know how students would have the LaunchCode/Unit1 or javascript-web-dev-projects directories at this point.